show/main.py

```diff
@@ -2028,6 +2028,29 @@ def ztp(status, verbose):
        cmd = cmd + " --verbose"
     run_command(cmd, display_cmd=verbose)
 
+#
+# 'static anycast gateway' command ("show static-anycast-gateway")
+#
+@cli.command('static-anycast-gateway')
+@clicommon.pass_db
+def sag(db):
+    """Show static anycast gateway information"""
+    sag_entry = db.cfgdb.get_entry('SAG', 'GLOBAL')
+    sag_mac = sag_entry.get('gateway_mac')
+    
+    header = ['MacAddress', 'Interfaces']
+    body = []
+    intf_dict = db.cfgdb.get_table('VLAN_INTERFACE')
+    if intf_dict:
+        for intf in intf_dict:
```

Comment:
It would be more brief to use
```python
for key,value in intf_dict.items():
    if value.get('static_anycast_gateway') == 'true':
```

Reply:
This is removed in the updated PR https://github.com/sonic-net/sonic-utilities/pull/2881